dataServices.js

```diff
@@ -0,0 +1,46 @@
+'use strict';
+
+const products = require('./src/data/products');
+const categories = require('./src/data/categories');
+
+let productMap = null;
+let categoryMap = null;
+let combinedProductMap = null;
+
+module.exports = {
+  getProducts: () => {
```

Comment:
Can you create these as a function, and add methods comments also. 
then export here.